### PR TITLE
feat: homepage

### PR DESCRIPTION
**Pull request by pankaj-prog**

### CODE REVIEW

File: hero.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  filter: brightness(15%);
+}
+
+/* content */
+.hero-content {
+  max-width: 600px;
```
Comment (ashishkatiyar2404) [inline hero.css]: use other units if, >10px.

File: hero.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 1rem;
+  z-index: 1;
+  color: var(--LIGHT-GRAY);
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
```
Comment (ashishkatiyar2404) [inline hero.css:53]: we can only use left and right at once with top /bottom. (either use left or right w/ top/bottom)

File: hero.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+.hero-header-title {
+  border-bottom: 1px solid white;
+}
+
+.hero-header-ctas i {
+  padding: 0.4rem;
+  border-radius: 8px;
+  border: 1px solid #ccc;
```
Comment (dhimanaishwarya) [inline hero.css]: you can add color variable here .

File: index.html [hunk 4]
```diff
@@ -4,14 +4,37 @@
...
+    <link rel="stylesheet" href="./hero.css" />
     <title>UI Library</title>
   </head>
   <body>
-    <div class="site">
-      <header class="site-header">
-        <div class="site-name">UI Library</div>
-      </header>
+    <main class="hero">
+  <img src="./src/assets/hero-background.webp" class="hero-background"></img>
```
Comment (dhimanaishwarya) [inline index.html]: you can add alt for img

File: hero.css [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 1;
+  padding: 0.5rem;
+}
+
+.hero-footer a {
+  text-decoration: none;
+  color: white;
```
Comment (ashishkatiyar2404) [inline hero.css]: use color variable😁😁

**Review by dhimanaishwarya — COMMENTED** [review]
Home Page is looking great!

**Review by ashishkatiyar2404 — CHANGES_REQUESTED** [review]

**Comment by ashishkatiyar2404** [comment]
the index page is really awesome with a dark theme🔥
Your code looks good to merge. Very efficient code and request some changes and then ready to merge.💯

File: hero.css (hunk 2 above)
Comment (pankaj-prog) [reply hero.css:53]: did that to take the full width

File: hero.css (hunk 3 above)
Comment (pankaj-prog) [reply hero.css]: updated

File: index.html (hunk 4 above)
Comment (pankaj-prog) [reply index.html]: updated

**Comment by SinchanaMS** [comment]
The homepage looks great Pankaj! Looks good to merge.

File: hero.css (hunk 5 above)
Comment (pankaj-prog) [reply hero.css]: updated

File: hero.css (hunk 1 above)
Comment (pankaj-prog) [reply hero.css]: sure

File: hero.css [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+  width: 100%;
+  height: 100%;
+  filter: brightness(15%);
+}
+
+/* content */
+.hero-content {
+  max-width: 37.5rem;
+  z-index: 1;
+  color: var(--WHITE);
```
Comment (deekshithmd) [inline hero.css:28]: better to give naming like bg-white, text-white like this rather than giving direct name of the color

File: hero.css [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+  align-items: center;
+  justify-content: space-between;
+  gap: 1rem;
+  z-index: 1;
+  color: var(--LIGHT-GRAY);
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  padding: 0.5rem 3vw;
```
Comment (deekshithmd) [inline hero.css:54]: here padding is given in vw is it required to be in vw?

File: hero.css [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+.hero-header-title {
+  border-bottom: 1px solid var(--WHITE);
+}
+
+.hero-header-ctas i {
```
Comment (deekshithmd) [inline hero.css:61]: I think it will look good if you increase size of icon for small extent

File: hero.css (hunk 7 above)
Comment (pankaj-prog) [reply hero.css:54]: yess

File: hero.css (hunk 8 above)
Comment (pankaj-prog) [reply hero.css:61]: okay

**Pull request merged**